completion/available/gradle.completion.bash

```diff
@@ -206,15 +212,17 @@ __gradle-generate-tasks-cache() {
 
 	# subproject tasks can be referenced implicitly from root project
 	if [[ $GRADLE_COMPLETION_UNQUALIFIED_TASKS == "true" ]]; then
-		local -a implicit_tasks=()
-		implicit_tasks=($(comm -23 <(printf "%s\n" "${subproject_tasks[@]}" | sort) <(printf "%s\n" "${root_tasks[@]}" | sort)))
+		local -a implicit_tasks
+		while IFS='' read -r line; do
+			implicit_tasks+=("$line")
+		done < <(comm -23 <(printf "%s\n" "${subproject_tasks[@]}" | sort) <(printf "%s\n" "${root_tasks[@]}" | sort))
 		for task in $(printf "%s\n" "${implicit_tasks[@]}"); do
```

Comment:
```suggestion
		for task in "${implicit_tasks[@]}"; do
```

Reply:
does that work the same even if the array has multi-word strings for values? seems right, I'll go with it.

Reply:
Yeah, I wasn't explicit, but this should behave the same because `IFS=$'\n'` is set in this context.